src/main/webapp/search-style.css

```diff
@@ -26,7 +26,7 @@ textarea, select, fieldset {
   width: 50%;
 }
 
-#search-button, #logout-button {
+#search-button, #logout-button, #private-logout-button, #add-to-search, #search-to-add, #add-button {
```

Comment:
This is a complete suggestion but you could just make this a class like "custom-button" and just give all of them the same class that way if you want to override one specifically you could do it through the ID

Reply:
This is a good suggestion, though I'd add that you should make the class something more descriptive than just "custom-button".